optuna/multi_objective/_hypervolume/utils.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional
+
+import numpy as np
+
+
+def _compute_2points_volume(
+    point1: np.ndarray, point2: np.ndarray, dim_bound: Optional[int] = None
```

Comment:
A comment not particularly for this function but it's probably cleaner to remove `dim_bound` until we actually need it.